pkgs/tools/misc/opentsdb/default.nix

```diff
@@ -1,34 +1,228 @@
-{ lib, stdenv, autoconf, automake, curl, fetchurl, fetchpatch, jdk8, makeWrapper, nettools
-, python3, git
+{ lib
+, stdenv
+, autoconf
+, automake
+, bash
+, curl
+, fetchFromGitHub
+, fetchurl
+, git
+, jdk8
+, makeWrapper
+, nettools
+, python3
 }:
 
-let jdk = jdk8; jre = jdk8.jre; in
-
-stdenv.mkDerivation rec {
+let
+  jdk = jdk8;
+  jre = jdk8.jre;
+  include = {
```

Comment:
Maybe it would be preferable to use `fetchMavenArtifact` or URLs using `mirror://maven/` instead of the direct `https://repo1.maven.org/maven2/`